src/main/kotlin/io/livekit/server/SipService.kt

```diff
@@ -90,4 +91,12 @@ interface SipService {
         @Body request: LivekitSip.CreateSIPParticipantRequest,
         @Header("Authorization") authorization: String
     ): Call<LivekitSip.SIPParticipantInfo>
+
+    @Headers("Content-Type: application/protobuf")
+    @POST("/twirp/livekit.SIP/TransferSIPParticipant")
+    fun transferSipParticipant(
+        @Body request: LivekitSip.TransferSIPParticipantRequest,
+        @Header("Authorization") authorization: String
+    ): Call<Empty>
```

Comment:
Does `Call<Void?>` work here? Keeps it consistent with the other empty response service calls.